chef/cookbooks/tempest/recipes/config.rb

```diff
@@ -357,7 +359,23 @@
     --is-public True \
     --page-size 1 \
     2> /dev/null | tail -n 2 | head -n 1 | awk '{ print $2 }')
-[ -n "$DOCKER_IMAGE_ID" ] && echo "$DOCKER_IMAGE_ID" > #{docker_image_id_file}
+if [ -n "$DOCKER_IMAGE_ID" ] ; then
+  echo "$DOCKER_IMAGE_ID" > #{docker_image_id_file}
+else
+  for node in #{docker_nodes.join(" ")}; do
+      ssh $node docker pull cirros
```

Comment:
Oh, that we probably don't have. But cirros image needs to be present on docker nodes.

So maybe again, it is wrong to do it in tempest barclamp and we should rely on mkcloud/other deployment solutions